writing-apps/creating-layouts.md

```diff
@@ -1,66 +1,55 @@
 # Creating Layouts
 
-Now that you know how to code, build, and package an app using Vala, Gtk, Meson, and Flatpak, it’s time to learn a little bit more about how to build out your app into something really useful. The first thing we need to learn is how to lay out widgets in our window. But we have a fundamental problem: We can only add one widget \(one “child”\) to `Gtk.Window`. So how do we get around that to create complex layouts in a Window? We have to add a widget that can contain multiple children. One of those widgets is `Gtk.Grid`.
+Now that you know how to code, build, and package an app using Vala, Gtk, Meson, and Flatpak, it’s time to learn a little bit more about how to build out your app into something really useful. The first thing we need to learn is how to lay out widgets in our window. But we have a fundamental problem: We can only add one widget \(one “child”\) to `Gtk.Window`. So how do we get around that to create complex layouts in a Window? We have to add a widget that can contain multiple children. One of those widgets is `Gtk.Box`.
 
-## Widgets Subclass Other Widgets
```

Comment:
Widget hierarchy in Gtk4 is weird. There is no Gtk.Container anymore, so I removed this section altogether. If you can explain widget hierarchy in Gtk4 then we can restore this section.

Reply:
Yeah I agree with removing for now. Things have changed big time there